popup/img/icon/clipboard.svg

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
+ "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
+ width="212.000000pt" height="237.000000pt" viewBox="0 0 212.000000 237.000000"
+ preserveAspectRatio="xMidYMid meet">
+<metadata>
+Created by potrace 1.15, written by Peter Selinger 2001-2017
+</metadata>
+<g transform="translate(0.000000,237.000000) scale(0.100000,-0.100000)"
+fill="#000000" stroke="none">
+<path d="M857 2259 c-62 -15 -112 -46 -156 -95 -38 -44 -71 -120 -71 -166 0
```

Comment:
Just curious, is this icon copyright free or created under creative commons? Otherwise, we will need to source it from somewhere else. I've been using [Rivolicons](http://rivolicons.hadrien.co/free/), but I don't see a clipboard icon there.

Reply:
The icon was from [here](https://github.com/vmware/clarity/issues/1860) but it may not be proper to use directly. So I found another copy to clipboard icon from [here](https://clarity.design/icons) and it was under MIT license